mrn_mysql_compat.h

```diff
@@ -962,3 +962,31 @@ typedef uint mrn_srid;
 #  define MRN_SET_MESSAGE_FROM_CTX(ctx, error_code)     \
   my_message(error_code, ctx->errbuf, MYF(0))
 #endif
+
+#if defined(MRN_MARIADB_P) &&                                      \
+    ((MYSQL_VERSION_ID >= 100526 && MYSQL_VERSION_ID < 100600) ||  \
+     (MYSQL_VERSION_ID >= 100619 && MYSQL_VERSION_ID < 100700) ||  \
+     (MYSQL_VERSION_ID >= 101109 && MYSQL_VERSION_ID < 101200))
+
+#  define MRN_GET_TABLE_NAME(query_tables)                         \
+  (query_tables->get_table_name().str)
+#  define MRN_GET_TABLE_LENGTH(query_tables)                       \
+  (query_tables->get_table_name().length)
+#else
+#  define MRN_GET_TABLE_NAME(query_tables)                         \
+  (query_tables->get_table_name())
+#  define MRN_GET_TABLE_LENGTH(query_tables)                       \
+  (strlen(query_tables->get_table_name()))
+#endif
+
+#if defined(MRN_MARIADB_P) &&                                      \
+    ((MYSQL_VERSION_ID >= 100526 && MYSQL_VERSION_ID < 100600) ||  \
+     (MYSQL_VERSION_ID >= 100619 && MYSQL_VERSION_ID < 100700) ||  \
+     (MYSQL_VERSION_ID >= 101109 && MYSQL_VERSION_ID < 101200))
```

Comment:
The conditions are the same, is it better to separate them?

Reply:
Ah...
I see. The conditions does not need to separate.
I concat them.

Reply:
I concatenated them.